### PR TITLE
Structure & Navbar

### PR DESCRIPTION
**Pull request by jandcreative**
1. React Router Library
2. Navbar Component
3. Update general CSS

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,38 +1,14 @@
...
-  }
-  to {
-    transform: rotate(360deg);
-  }
+:root {
+    --text: #3A3938;
+}
+
+body {
+  font-family: 'Rubik',sans-serif;
```
Comment (jelin-mi) [inline src/App.css:6]: Add a space after a comma, thanks.

File: src/App.js [hunk 2]
```diff
@@ -1,11 +1,22 @@
 import "./App.css";
+import { Routes, Route } from "react-router-dom";
+import Home from "./pages/Home/Home.jsx";
+import Tracks from "./pages/Tracks/Tracks.jsx";
+import Bikers from "./pages/Bikers/Bikers.jsx";
+import Error from "./pages/Error/Error.jsx";
+import Navbar from "./components/Navbar/Navbar";
+import './Reset.css';
```
Comment (jelin-mi) [inline src/App.js:8]: Just an idea, I would move App.css right above Reset.css in the line 7.

File: src/App.css (hunk 1 above)
Comment (jelin-mi) [inline src/App.css:6]:
```suggestion
  font-family: 'Rubik', sans-serif;
```
Add a space after a comma, thanks.

File: src/App.js [hunk 3]
```diff
@@ -1,11 +1,22 @@
 import "./App.css";
```
Comment (jelin-mi) [inline src/App.js:1]: Do not add empty files as those are not uploaded and it causes errors. Or maybe you forgot to push your last changes?

> Failed to compile.
> 
> Module not found: Error: Can't resolve './pages/Home/Home.jsx' in '/Users/michaela/code/pirinexus-route/pirinexus-route/src'
> ERROR in ./src/App.js 6:0-41
> Module not found: Error: Can't resolve './pages/Home/Home.jsx' in '/Users/michaela/code/pirinexus-route/pirinexus-route/src'
> 
> ERROR in ./src/App.js 7:0-47
> Module not found: Error: Can't resolve './pages/Tracks/Tracks.jsx' in '/Users/michaela/code/pirinexus-route/pirinexus-route/src'
> 
> ERROR in ./src/App.js 8:0-47
> Module not found: Error: Can't resolve './pages/Bikers/Bikers.jsx' in '/Users/michaela/code/pirinexus-route/pirinexus-route/src'
> 
> ERROR in ./src/App.js 9:0-44
> Module not found: Error: Can't resolve './pages/Error/Error.jsx' in '/Users/michaela/code/pirinexus-route/pirinexus-route/src'
> 
> webpack compiled with 4 errors

File: src/components/Navbar/Navbar.css [hunk 4]
```diff
@@ -0,0 +1,28 @@
+/* Navbar */
```
Comment (jelin-mi) [inline src/components/Navbar/Navbar.css:1]: Run `Format Document` before pushing your code, please.

File: src/components/Navbar/Navbar.jsx [hunk 5]
```diff
@@ -0,0 +1,23 @@
+import { NavLink } from "react-router-dom";
```
Comment (jelin-mi) [inline src/components/Navbar/Navbar.jsx:1]: Run `Format Document` before pushing your code, please.

**Review by jelin-mi — CHANGES_REQUESTED** [review]
Please adjust the code as per my comments, just minor things.
Thanks!

File: src/components/Navbar/Navbar.jsx (hunk 5 above)
Comment (jandcreative) [reply src/components/Navbar/Navbar.jsx:1]: Corrected

File: src/App.css (hunk 1 above)
Comment (jandcreative) [reply src/App.css]: Corrected!

**Review by jelin-mi — APPROVED** [review]
Ready to be merged to `develop`!

**Pull request merged**